docs/src/pages/components/cdk-components/number-input/examples/customErrorsNumberInput.js

```diff
@@ -4,29 +4,30 @@ import { useState } from "react";
 const code = `() => {
   const [value, setValue] = useState("");
   const [errorMessage, setErrorMessage] = useState("");
-  const onChange = ({ value }) => {
+  const onChange = ({ value, error }) => {
```

Comment:
I find this example a bit difficult to understand, perhaps it is better to separate the two cases.

Reply:
I agree, but this is example is in every other input and textarea component. Since we have to review the error prop and how we handle errors in the documentation (empty string issue), we can discuss this there. But. for now, I think I prefer consistency rather than a better example...

Reply:
I agree. 👍🏼